[ignore]
unit = [
	"tests/unit/modules/test_boto3_elasticsearch.py",
	"tests/unit/modules/test_boto3_route53.py",
	"tests/unit/modules/test_boto_route53.py",
	"tests/unit/utils/test_boto3mod.py",
]
integration = [
]
[skip]
unit = [
	"tests/pytests/unit/grains/test_core.py::test_get_server_id",
	"tests/pytests/unit/modules/test_rpm_lowpkg.py::test_version_cmp_rpm_all_libraries[HAS_RPM]",
	# Requires Python 3.7. Non-bundle client uses Python 3.6, which doesn't have blowfish algorithm
	"tests/pytests/unit/utils/test_pycrypto.py::test_gen_hash_crypt[blowfish-expected2]",
	"tests/unit/cli/test_support.py::ProfileIntegrityTestCase::test_jobs_trace_template_profile",
	"tests/unit/cli/test_support.py::ProfileIntegrityTestCase::test_non_template_profiles_parseable",
	"tests/unit/cli/test_support.py::ProfileIntegrityTestCase::test_users_template_profile",
	"tests/unit/modules/test_dpkg_lowpkg.py::DpkgTestCase::test_info",
	# Upstream removed vendored Tornado in https://github.com/saltstack/salt/commit/4baea1a97be0389fabe5307d084579134a1f9b7a
	"tests/unit/transport/test_ipc.py::IPCMessagePubSubCase::test_async_reading_streamclosederror",
	"tests/unit/transport/test_ipc.py::IPCMessagePubSubCase::test_multi_client_reading",
]
integration = [
	"tests/integration/cli/test_custom_module.py::SSHCustomModuleTest::test_ssh_custom_module",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/cli/test_custom_module.py::SSHCustomModuleTest::test_ssh_sls_with_custom_module",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/loader/test_ext_modules.py",  # Make pytest to stuck in sle15sp1 classic pkg
	"tests/integration/modules/test_pip.py::PipModuleTest::test_chained_requirements__non_absolute_file_path",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_issue_2087_missing_pip",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_issue_4805_nested_requirements",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_pip_install_multiple_editables_and_pkgs",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_requirements_as_list__non_absolute_file_path",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_requirements_as_list_of_chains__cwd_not_set__absolute_file_path",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_pip.py::PipModuleTest::test_requirements_as_list_of_chains__cwd_set__absolute_file_path",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_tls.py::TLSModuleTest::test_revoked_cert_should_return_False_from_validate",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_tls.py::TLSModuleTest::test_validating_revoked_cert_with_no_crl_file_should_return_False",  # Needs investigation: Fails only with Salt Bundle
	"tests/integration/modules/test_tls.py::TLSModuleTest::test_with_existing_ca_signing_csr_should_produce_valid_cert",  # Needs investigation: Fails only with Salt Bundle
	# Tests related to pygit2 are failing for Classic Salt in Leap 15.5. This needs further research.
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_all_saltenvs_base",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_base",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_fallback",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_includes_disabled",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_includes_enabled",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_includes_enabled_solves___env___with_mountpoint",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_mountpoint_parameter",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_multiple_sources_dev_master_merge_lists",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_multiple_sources_dev_master_no_merge_lists",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_multiple_sources_master_dev_merge_lists",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_multiple_sources_master_dev_no_merge_lists",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_multiple_sources_with_pillarenv",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_root_and_mountpoint_parameters",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_root_parameter",
	"tests/integration/pillar/test_git_pillar.py::TestPygit2SSH::test_single_source",
]
functional = [
	"tests/pytests/functional/channel/test_server.py::test_pub_server_channel[transport(zeromq)]",
	"tests/pytests/functional/modules/test_dockermod.py::test_docker_call",  # Needs investigation: Fails only with Salt Bundle - salt.exceptions.CommandExecutionError: /var/tmp/venv-salt-minion/bin/python.original: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /var/tmp/venv-salt-min...
	"tests/pytests/functional/modules/test_dockermod.py::test_docker_highstate",  # Needs investigation: Fails only with Salt Bundle - salt.exceptions.CommandExecutionError: /var/tmp/venv-salt-minion/bin/python.original: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /var/tmp/venv-salt-min...
	"tests/pytests/functional/modules/test_dockermod.py::test_docker_sls",  # Needs investigation: Fails only with Salt Bundle - salt.exceptions.CommandExecutionError: /var/tmp/venv-salt-minion/bin/python.original: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /var/tmp/venv-salt-min...
	"tests/pytests/functional/runners/test_winrepo.py::test_legacy_update_git_repos",
	"tests/pytests/functional/runners/test_winrepo.py::test_update_git_repos",
	"tests/pytests/functional/states/test_pip_state.py::test_pip_installed_name_test_mode",
	"tests/pytests/functional/states/test_pip_state.py::test_pip_installed_pkgs_test_mode",
	"tests/pytests/functional/transport/ipc/test_pub_server_channel.py::test_issue_36469_tcp",  # Make pytest to hang on sle15sp1
	"tests/pytests/functional/transport/ipc/test_pub_server_channel.py::test_publish_to_pubserv_ipc",  # Make pytest to hang during cleanup
	"tests/pytests/functional/transport/tcp/test_message_client.py::test_message_client_reconnect",
	"tests/pytests/functional/transport/zeromq/test_pub_server_channel.py::test_zeromq_filtering",  # Make pytest to hang on sle15sp1
	# We use python3-yamllint-1.22.1-150500.3.5.1.noarch, upstream hardcodes version 1.26.3 or greater
	"tests/pytests/functional/utils/yamllint/test_yamllint.py::test_version",
	"tests/pytests/functional/states/pkgrepo/test_debian.py::test_adding_repo_file_arch", # Seems to cause "Socket exception: Connection timed out" on Debian 11
]
scenarios = [
	"tests/pytests/scenarios/blackout/test_minion_blackout.py::test_blackout",
	"tests/pytests/scenarios/blackout/test_minion_blackout.py::test_blackout_nonwhitelist",
	"tests/pytests/scenarios/blackout/test_minion_blackout.py::test_blackout_whitelist",
	"tests/pytests/scenarios/multimaster/beacons/test_inotify.py::test_beacons_duplicate_53344",
	"tests/pytests/scenarios/reauth/test_reauth.py::test_reauth",
]
